app/pages/sections/catalogue-solutions/select-solution/template.njk

```diff
@@ -19,6 +20,12 @@
         <form method="post">
           <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
 
+          {{
+            radiobuttonOptions({
+              question: questions[0]
```

Comment:
if there is only 1 question I would get rid of the array and just pass it straight in

Reply:
it's how we have done it in description and supplier and pretty much everywhere. I think we should leave this as is for now.

Reply:
ok